### DISCUSSION TITLE
Android app in background, drops connection while receiving files

### DISCUSSION THREAD

**Discussion by thedamnatus** [localsend/localsend#1399] [Discussion (English, International)]
I just installed localsend to my windows pc and to our android (samsung) phones. 
Tried sending a couple of large (about 1gb each) files from windows to the phones as a test - in both cases, when the localsend app is on the phone screen, it works flawlessly. 
However, as soon as the localsend app is in background and I do something else on my phone (web browsing, other apps, etc) - at first the transfer speed starts to degrade (from 20 mbps to 5 mbps), then localsend drops a file outright (with a network error, something like "remote host forcibly closed the connection"), switches to transferring the next one, after some time it drops it as well, and so on.
Is this intended? Is localhost able to receive files on android only when the app is active? It is whitelisted from any sleep modes or power saving modes, etc.

**Comment by mystyc** [comment]
Yeah, I have the same issue.  The longest screen timeout setting on my device is 10 min.  At transfer rates of around 1 to 2 MB/s, a 1 GB file takes about 8 to 17 minutes to finish, and so the transfer ends with an error.  
The app otherwise runs smoothly with an almost set-and-forget style, except for large files.  
I guess most users on here either only transfer small files or possibly don't realize that this is the actual cause of their submitted problem.
